presto-parser/src/main/java/com/facebook/presto/sql/tree/Identifier.java

```diff
@@ -87,12 +87,12 @@ public boolean equals(Object o)
         }
 
         Identifier that = (Identifier) o;
-        return Objects.equals(value, that.value);
```

Comment:
All, I totally agree. I just am afraid this change will introduce regression(s), because we lack proper equivalence in certain places.
For example `select a from (values 1) t(a) group by "a"` works before/after this change, but query `select a + 1 from (values 1) t(a) group by "a" + 1` will _start_ to fail (`'(a + 1)' must be an aggregate expression or appear in GROUP BY clause`).